src/app/components/resources-and-support/frequently-asked-questions.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import envConfig from '../../../envConfig';
+import { ExternalLink } from '../../../shared';
+
+const tableData = {
+  'If I am an EHR, how can I submit MIPS Quality data for my clients?': `
+    <ul>
+      <li>
+        Submissions UI: EHRs can submit via the Submissions UI by logging into the QPP site <a href='https://qpp.cms.gov' target='_blank' rel='noopener noreferrer'>qpp.cms.gov</a>. They will have to be connected as a Staff user for each of the Individuals and Groups they need to submit quality data for. Submissions for each individual or group could be uploaded as an excel file for efficiency.
+      </li>
+      <li>
+        Burden Reduction: EHRs may apply for Burden Reduction which allows them to obtain a registry token to use to submit via the Submissions API. See the process outline in this <a href='https://groups.google.com/forum/#!msg/qpp-apis/EKqpKejepwY/uPrgllFiDQAJ' target='_blank' rel='noopener noreferrer'>Google Group post</a>
+      </li>
+    </ul>
+  `,
+  'What is the performance threshold for the Submissions API?': '25 rps avg threshold, 40 rps burst threshold.',
+};
+
+const FrequentlyAskedQuestions = () => {
+  return (
+    <>
+      <h2 className='ds-h2'>Frequently Asked Questions</h2>
+
+      <h3 className='ds-h3'>General</h3>
+      <p>
+        See <ExternalLink href={`${envConfig.cmsGithubIo}/qpp-developer-preview-docs/frequently-asked-questions`} text='Developer Preview Frequently Asked Questions' />
```

Comment:
looks like it was already like this pre-upgrade, but should both of these sections point at the dev preview docs?

Reply:
ill confirm with Michelle

Reply:
confirmed with Michelle